src/pages/common/BaseWrapper.py

```diff
@@ -87,8 +86,3 @@ def go_to_site(self):
             return site
         except TimeoutError:
             print("Site downloading failed, timeout")
-
-
-    def scroll_down(self, locator, timeout=10):
```

Comment:
I didn't see any usage of this method. Is it necessary to make some elements to appear/load by scrolling?